client/jetpack-cloud/sections/partner-portal/license-bundle-card/index.tsx

```diff
@@ -16,12 +16,13 @@ interface Props {
 	tabIndex: number;
 	product: APIProductFamilyProduct;
 	onSelectProduct: ( value: APIProductFamilyProduct ) => void | null;
+	showLicenseInfoModal: boolean;
 }
 
 export default function LicenseBundleCard( props: Props ) {
 	const { tabIndex, product, onSelectProduct } = props;
 	const productTitle = getProductTitle( product.name );
-	const [ showLightbox, setShowLightbox ] = useState( false );
+	const [ showLightbox, setShowLightbox ] = useState( props.showLicenseInfoModal );
```

Comment:
When we open a modal, we should also update the URL so the user can easily share the correct modal link. wdyt?

Reply:
That is a very good point, @jkguidaven. I have made the changes here - https://github.com/Automattic/wp-calypso/pull/78850/commits/9a44723c4e9364c9294c3f63689b1e1d34bdb738